popup/warehouse.js

```diff
@@ -55,27 +59,58 @@ function createList(storeObj) {
 				// });
 			});
 
+			accButton.addEventListener("keydown" , event => {
+				if (accButton.textContent.length > 30) {
```

Comment:
From the code written, I understand that it is supposed to not take in any new characters after the 30 char limit. But when I tested it, the new character entered stays, while the old character is removed and the cursor suddenly jumps to the starting point of the text.   
I think a better way to implement would be to take in the string of any length, but before storing, (in the `persist` function) call a function which trims the given string to the cutOff length. A reference implementation can be see [here](https://github.com/allen505/save-for-later/blob/0422715a00253fe4581b6098ecb8709ece688b8e/popup/warehouse.js#L89).  
If you could make the referenced line into a seperate function which can be called in the referenced area and in the `persist` function it would be great.

Reply:
The problem is not when i am persisting longer strings, actually the problem is about the visualization during the input, that after 30 characters it starts messing with the buttons, please try by extending the value in the condition and let me know how it behaves for you.

This is what is happening to me when i allow longer strings:

![image](https://user-images.githubusercontent.com/849502/94206959-a11f0100-fe94-11ea-8020-5cd74e0e67ba.png)

I guess it can be solved with stiles and an extra DIV as well

Reply:
I figured out something else, while testing in Chrome, it does not accept spaces in the title text and the max length before starting moving down the buttons is 19 characters.

Is it same for you? 

In Firefox, it does accept spaces, but the max length is 25 characters (it is not 30)